st2actions/st2actions/query/mistral/v2.py

```diff
@@ -74,7 +74,7 @@ def query(self, execution_id, query_context):
             raise
 
         try:
-            tasks = self._get_workflow_tasks(mistral_exec_id)
+            result['extra']['tasks'] = self._get_workflow_tasks(mistral_exec_id)
```

Comment:
Slightly confused - why do we also put this in extra?

I believe this will break existing "unified" tasks format.